osquery/tables/system/posix/tests/known_hosts_tests.cpp

```diff
@@ -0,0 +1,71 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <fstream>
+
+#include <boost/filesystem.hpp>
+#include <gtest/gtest.h>
+
+#include <osquery/core/scope_guard.h>
+
+#include "osquery/tables/system/posix/known_hosts.h"
+
+namespace osquery {
+namespace tables {
+
+class KnownHostsImplTests : public testing::Test {};
+
+TEST_F(KnownHostsImplTests, get_from_file) {
+  namespace fs = boost::filesystem;
+
+  auto results = QueryData{};
+  auto directory = fs::temp_directory_path() /
+                   fs::unique_path("osquery.known_hosts_impl_tests.%%%%-%%%%");
+
+  ASSERT_TRUE(fs::create_directories(directory));
+
+  auto const path_guard =
+      scope_guard::create([directory]() { fs::remove_all(directory); });
```

Comment:
SetUp 
TearDown
Can be used for that.

Reply:
Yes, I know. But `SetUp/TearDown` do it for all tests in the file, but I need it only for one. Of course we have only one test and it does not make a lot of sense, but eventually we will add more and it make a big mess in `SetUp/TearDown`.
Therefore, if you don't have a strict objections I'd like to leave it like this. What do you think?